src/protobuf-net.Grpc.AspNetCore/ServicesExtensions.cs

```diff
@@ -30,6 +31,21 @@ public static IGrpcServerBuilder AddCodeFirstGrpc(this IServiceCollection servic
             return builder;
         }
 
+        public static IGrpcServerBuilder WithBinderConfiguration(this IGrpcServerBuilder builder, Action<BinderOptions> binderOptions)
+        {
+            var options = new BinderOptions();
+            binderOptions(options);
+            builder.Services.TryAddSingleton(BinderConfiguration.Create(options.MarshallerFactories, options.ServiceBinder));
+            return builder;
+        }
+
+        public sealed class BinderOptions
```

Comment:
the public API should have intellisense; if this didn't raise a warning, that's a bug that I should fix

Reply:
I was building in `Debug` so it wasn't flagging them locally and I didn't review CI beyond seeing it passed. Maybe `TreatWarningsAsErrors` should/could be turned on?